ci-operator/config/redhat-developer/odo/redhat-developer-odo-main__odo-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make bin
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/build-root/Dockerfile
+images:
+- context_dir: openshift-ci/build-root/source-image
```

Comment:
How are different versions of ODO being tested here?

Or maybe the question we should start with is, what version of ODO is being tested?

Reply:
Branch main is V3